src/main/java/duke/exceptions/EmptyBodyException.java

```diff
@@ -0,0 +1,13 @@
+package duke.exceptions;
+
+/**
+ * Exception thrown when the command body is empty.
+ */
+public class EmptyBodyException extends BaseException {
```

Comment:
name of the exception could be more descriptive here. instead of using "empty" could use "missing" to show that body cannot be empty. other exceptions follow this vocabulary by using words such as "invalid" or "illegal"